### PR TITLE
[PfemFluid] cleaning of elements

### PR DESCRIPTION
**Pull request by AFranci**
Cleaning of elements avoiding code duplication and removing unused modules

### CODE REVIEW

File: applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_fluid_element.cpp [hunk 1]
```diff
@@ -565,14 +519,71 @@ namespace Kratos
...
+  {
+    KRATOS_TRY;
+
+    if (this->GetProperties().Has(YIELD_SHEAR) && this->Has(YIELDED))
+    {
+      double tolerance = 1e-10;
+      if (this->GetProperties()[YIELD_SHEAR] > tolerance)
+      {
+
+        double TauNorm = sqrt(2.0 * rElementalVariables.UpdatedDeviatoricCauchyStress[0] * rElementalVariables.UpdatedDeviatoricCauchyStress[0] +
```
Comment (AlejandroCornejo) [inline applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_fluid_element.cpp]: const?

File: applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_fluid_element.cpp [hunk 2]
```diff
@@ -565,14 +519,71 @@ namespace Kratos
...
+  }
+
+  template <>
+  void TwoStepUpdatedLagrangianVPImplicitFluidElement<3>::SetYieldedElements(ElementalVariables &rElementalVariables)
+  {
+    KRATOS_TRY;
+
+    if (this->GetProperties().Has(YIELD_SHEAR) && this->Has(YIELDED))
+    {
+      double tolerance = 1e-10;
```
Comment (AlejandroCornejo) [inline applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_fluid_element.cpp]: const?

File: applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_solid_element.cpp [hunk 3]
```diff
@@ -327,6 +327,52 @@ namespace Kratos
...
+    rElementalVariables.UpdatedDeviatoricCauchyStress[0] = rElementalVariables.CurrentDeviatoricCauchyStress[0];
+    rElementalVariables.UpdatedDeviatoricCauchyStress[1] = rElementalVariables.CurrentDeviatoricCauchyStress[1];
+    rElementalVariables.UpdatedDeviatoricCauchyStress[2] = rElementalVariables.CurrentDeviatoricCauchyStress[2];
+    rElementalVariables.UpdatedDeviatoricCauchyStress[3] = rElementalVariables.CurrentDeviatoricCauchyStress[3];
+    rElementalVariables.UpdatedDeviatoricCauchyStress[4] = rElementalVariables.CurrentDeviatoricCauchyStress[4];
+    rElementalVariables.UpdatedDeviatoricCauchyStress[5] = rElementalVariables.CurrentDeviatoricCauchyStress[5];
+
+    rElementalVariables.UpdatedTotalCauchyStress[0] =
+        rElementalVariables.CurrentDeviatoricCauchyStress[0] + rElementalVariables.MeanPressure;
+    rElementalVariables.UpdatedTotalCauchyStress[1] =
```
Comment (AlejandroCornejo) [inline applications/PfemFluidDynamicsApplication/custom_elements/two_step_updated_lagrangian_V_P_implicit_solid_element.cpp:364]: nice

File: applications/PfemFluidDynamicsApplication/custom_elements/updated_lagrangian_element.cpp [hunk 4]
```diff
@@ -1619,6 +1619,37 @@ namespace Kratos
     return std::sqrt(2.0 * NormS);
   }
 
+  template <>
+  void UpdatedLagrangianElement<2>::ComputeMechanicalDissipation(ElementalVariables &rElementalVariables)
+  {
+    KRATOS_TRY;
+    const double volumetric_strain = (rElementalVariables.SpatialDefRate[0] + rElementalVariables.SpatialDefRate[1]) * 0.5;
+
+    double mechanicalDissipation = rElementalVariables.UpdatedDeviatoricCauchyStress[0] * (rElementalVariables.SpatialDefRate[0] - volumetric_strain) +
```
Comment (AlejandroCornejo) [inline applications/PfemFluidDynamicsApplication/custom_elements/updated_lagrangian_element.cpp]: const

Comment (AlejandroCornejo) [reply applications/PfemFluidDynamicsApplication/custom_elements/updated_lagrangian_element.cpp]: usually we use mechanical_dissipation for this kind of local variables

Comment (AFranci) [reply applications/PfemFluidDynamicsApplication/custom_elements/updated_lagrangian_element.cpp]: done!

**Review by AlejandroCornejo — APPROVED** [review]

**Pull request merged**